integration/kubernetes/k8s-cpu-ns.bats

```diff
@@ -20,7 +20,7 @@ setup() {
 }
 
 @test "Check CPU constraints" {
-	wait_time=30
+	wait_time=120
```

Comment:
Should we just wait to see the pods status based on watching instead of sleep?  Or, WDYT?

Reply:
I think - if we can - yes, we should do sleepy polls across the whole codebase rather than hard wired 'sleep and pray' :-)
If we can make this a poll loop with a sleep and a timeout, then great, please do :-)

Reply:
This is what we do, you can check: 
https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L145-L158

It polls the status every 5 secs... so the maximum is 120 so that we can fail the test.

Reply:
@egernst and @grahamwhaley ^

Reply:
yep, understood - our mistake :-)